src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -331,17 +343,20 @@ export const Participants = () => {
         ),
         value: '',
```

Comment:
While you're here, I feel like this shouldn't be blank, in case the column is used for sorting:
```tsx
        value: status,
```